tools/dbslower.py

```diff
@@ -27,7 +27,7 @@
     dbslower postgres            # trace PostgreSQL queries slower than 1ms
     dbslower postgres -p 188 322 # trace specific PostgreSQL processes
     dbslower mysql -p 480 -m 30  # trace MySQL queries slower than 30ms
-    dbslower mysql -p 480 -v     # trace MySQL queries and print the BPF program
+    dbslower mysql -p 480 -v    # trace MySQL queries and print the BPF program
```

Comment:
Yes, but pep8 rejects this as its > 80 chars (81 exactly). I'm not sure what's the best solution here...?